### PR TITLE
Support Ubuntu 18.04

### PR DESCRIPTION
**Pull request by juhlee-microsoft**
1. Replaced with replacable xml parameters
2. Enable Ubuntu 18.04 support for Open MPI
3. Fixed the infinite loop of TestRunning
4. Removed kernel module verification. No longer needed.

### CODE REVIEW

File: Testscripts/Linux/SetupRDMA.sh [hunk 1]
```diff
@@ -132,28 +134,37 @@ function Main() {
             echo "* soft memlock unlimited" >> /etc/security/limits.conf
             echo "* hard memlock unlimited" >> /etc/security/limits.conf
             ;;
-        *)
-            LogErr "Distro '$DISTRO' not supported or not implemented"
-            SetTestStateFailed
-            exit 0
-            ;;
+		ubuntu*)
```
Comment (LiliDeng) [inline Testscripts/Linux/SetupRDMA.sh]: there are some indent inconsistent in this PR

Comment (juhlee-microsoft) [reply Testscripts/Linux/SetupRDMA.sh]: fixed in c54032af7b49da4bf748b41122c7ceeafc57c62b

File: Testscripts/Linux/SetupRDMA.sh [hunk 2]
```diff
@@ -11,9 +11,9 @@
 ########################################################################################################
 # Source utils.sh
 . utils.sh || {
-    echo "Error: unable to source utils.sh!"
-    echo "TestAborted" >state.txt
-    exit 0
+	echo "Error: unable to source utils.sh!"
```
Comment (iamshital) [inline Testscripts/Linux/SetupRDMA.sh]: This PR also has same indentation problem.

image

Comment (juhlee-microsoft) [reply Testscripts/Linux/SetupRDMA.sh]: You missed our internal communication. There are many files having space based indentation. Should use tab size 4 indentation in coding style. Will find a way to check indentation condition in PR.

Comment (juhlee-microsoft) [reply Testscripts/Linux/SetupRDMA.sh]: You missed our internal communication. There are many files having space based indentation. Should use tab size 4 indentation in coding style. Will find a way to check indentation condition in PR.

**Pull request closed without merging**